spec.yaml

```diff
@@ -1837,15 +1851,23 @@ paths:
 
         The data is aggregated but not converted to USD, so OHLC values are in the quote currency unit and Volume
         is in the base currency unit.
-        
-        Candles have the following history based on size:
 
-        * `1d`: Inception
-        * `4h`: 120 days
-        * `1h`: 30 days
-        * `30m`: 14 days
-        * `5m`: 3 days
-        * `1m`: 24 hours
+        History is available for all candle sizes from inception, but
+        there are limits on the length of the date range you can
+        request at a time:
+
+        | interval | limit    |
+        |----------|----------|
+        | `1d`     | no limit |
+        | `4h`     | 120 days |
+        | `1h`     | 30 days  |
+        | `30m`    | 14 days  |
+        | `5m`     | 3 days   |
+        | `1m`     | 1 day    |
+
+        If the difference between your `end` and `start` parameters
+        for the `interval` you're requesting exceeds the limit, then
+        you'll receive an error.
```

Comment:
Are aggregated market pair candles changed? I didn't realize you did that too.

Reply:
Yup:

* https://github.com/nomics-crypto/nommer/pull/411/files#diff-429152cf9bfb8b252921826411282bc9
* https://api.nomics.com/v1/markets/candles?base=BTC&interval=1h&key=REDACTED&quote=USD&start=2019-01-01T00:00:00Z&end=2019-01-31T00:00:00Z